vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml

```diff
@@ -63,6 +63,7 @@
                 <property name="skipResponseEntity" value="true" />
                 <property name="lastRequestInterceptor"><null/></property>
                 <property name="lastResponseInterceptor"><null/></property>
+                <property name="sslCertificateCheckEnabled" value="${http.resource-validator.ssl-certificate-check}" />
```

Comment:
`http.ssl.check-certificate` is not working?

Reply:
This property doesn't help :(

Reply:
That's weird